src/System.Management.Automation/engine/remoting/commands/PushRunspaceCommand.cs

```diff
@@ -172,6 +174,18 @@ public override PSCredential Credential
                    ParameterSetName = EnterPSSessionCommand.VMNameParameterSet)]
         public String ConfigurationName { get; set; }
 
+        #region Supress PSRemotingBaseCmdlet SSH hash parameter set
+
+        /// <summary>
+        /// Connection
```

Comment:
My comment was specific to the content of the comment on line 180. It is not clear. In other places you state, "Suppress Connection"